packages/sn-filter-pane/src/components/ListboxGrid/ListboxGrid.tsx

```diff
@@ -89,6 +91,13 @@ function ListboxGrid({ stores }: { stores: IStores }) {
     preventDefaultBehavior(event);
   };
 
+  const handleFocus = useCallback(() => {
+    // Reset focus on small devices due to auto focus.
+    if (sense?.isSmallDevice?.()) {
+      resetZoom();
+    }
+  }, [sense?.isSmallDevice?.()]);
```

Comment:
```suggestion
  }, [sense?.isSmallDevice]);
```

Reply:
Yeah I thought the same, but it is a callable function. We do that in other places, such as here: https://github.com/qlik-oss/sn-list-objects/blob/main/packages/sn-filter-pane/src/components/ListboxPopoverContainer.tsx#L85 

```
const isSmallDevice = sense?.isSmallDevice?.() ?? false;
```

Reply:
Or you meant in the listen array? 🤔 Will it execute if I put it like a plain function 🤷 (I don't know)